addons/mail/static/src/model/model_manager.js

```diff
@@ -147,7 +151,8 @@ export class ModelManager {
         /**
          * Create the messaging singleton record.
          */
-        this.models['mail.messaging'].create(values);
+        const messaging = this.models['mail.messaging'].insert(values);
+        Object.assign(odoo.__DEBUG__, { messaging });
```

Comment:
Not sure whether it's intentional to keep even in non-debug mode

Reply:
Yes it is intentional, it makes debugging much easier to always have it there.